spec/sections/11-geometry_extension.adoc

```diff
@@ -1632,6 +1634,23 @@ The function http://www.opengis.net/def/function/geosparql/length[`geof:length`]
 See the <<Recommendation for units of measure, Recommendation for units of measure>>.
 ====
 
+[%unnumbered]
+[source,turtle]
+----
+geof:latitude (geom: ogc:geomLiteral): xsd:double
+----
+
+The function http://www.opengis.net/def/function/geosparql/latitude[`geof:longitude`] returns the latitude of `geom` in meters. If the geometry is a point it returns the point's latitude. If the geometry is a more complex geometry it returns the latitude of its centroid as calculated by geof:centroid.
```

Comment:
in metrers -> in degree?

Reply:
How about "returns the latitude of `geom` in the unit that is proposed by the coordinate reference system associated with the geometry"?

Reply:
from my feeble understanding, lat and lon would only ever be defined as an angle (normal to the surface and plane of the celestial body resp. rotational angle between the zero meridian), and thus only degree (or radian??) would be suitable. The other question is related to #398 i.e. should a point defined on a metric (projected) CRS be eligible for automatic transformation into CRS:84 in order to determine its lat/lon on the celestial body or not